spec.bs

```diff
@@ -369,6 +369,15 @@ below and in [[#index-defined-elsewhere]].
 
     <div class="note">TODO: Bikeshed the name.</div>
 
+    <div class="note">To quickly support an initial SPC experiment,
+    this API was designed atop existing implementations of the Payment
+    Request and Payment Handler APIs. There is now general agreement
+    that SPC should be usable independent of Payment Request.  We
```

Comment:
Perhaps "There is not general agreement that using SPC independent of Payment Request **should be considered**" or "**should be explored**"? I don't want to make it sound like we have a concrete plan in place.